addons/mass_mailing/migrations/11.0.2.0/post-migration.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def map_list_id_to_list_ids(cr):
+    """Set the list_ids for the list_id in mail.mass_mailing.contact"""
+    cr.execute(
```

Comment:
This is the case where you should use openupgradelib `m2o_to_x2m` (https://doc.therp.nl/openupgrade/API.html#openupgradelib.openupgrade.m2o_to_x2m)